src/overlay_params.cpp

```diff
@@ -691,16 +719,19 @@ parse_overlay_config(struct overlay_params *params,
 
 #ifdef _WIN32
    params->toggle_hud = { VK_F12 };
+   params->toggle_preset = { VK_F5 };
```

Comment:
This should correspond to the x11 keybind and be F10

Reply:
I noticed this wasn't the case for some of the other keybinds so I just made it F5 since the last one was F4.  I'll change it to F10 though